newdle/client/src/styles/palette.scss

```diff
@@ -7,3 +7,5 @@ $busy: #ffb3b8;
 $green: #62f46f;
 $light-teal: #00caa8;
 $blue: #2185d0;
+$candidate: rgba(130, 130, 130, 0.3);
```

Comment:
pretty sure scss lets you define rgba colors in hex notation, e.g. `rgba(#rrggbb, 0.3)` or something like that